ansible/configs/ansible-middleware-workshop/files/cloud_providers/azure_cloud_template.j2

```diff
@@ -0,0 +1,428 @@
+            ],
+            "tags": {
+                "Name": "{{instancename}}",
+                "internaldns": "{{instancename}}.{{chomped_zone_internal_dns}}",
+                "owner": "{{ email | default('unknownuser') }}",
+                "Project": "{{project_tag}}",
+{% for tag in instance['tags'] %}
+                "{{tag['key']}}": "{{tag['value']}}",
+{% endfor %}
+                "{{project_tag}}": "{{ instance['name'] }}"
+            },
+            "properties" : {
+                "hardwareProfile" : {
+                    "vmSize" : "{{instance['flavor'][cloud_provider]}}"
+                },
+                "osProfile" : {
+                    "computerName" : "{{instancename}}",
+                    "adminUsername" : "[parameters('adminUsername')]",
+                    "linuxConfiguration" : {
+                        "disablePasswordAuthentication" : "true",
+                        "ssh" : {
+                            "publicKeys" : [
+                                {
+                                    "path" : "[variables('sshKeyPath')]",
+                                    "keyData" : "[parameters('sshKeyData')]"
+                                }
+                            ]
+                        }
+                    }
+                },
+                "storageProfile" : {
+                    "imageReference" : "[variables('rhel')]",
+                    "osDisk" : {
+                        "caching" : "ReadWrite",
+                        "name" : "{{instancename}}-osdisk",
+                        "createOption" : "FromImage",
+                        "diskSizeGB" : "{{instance['rootfs_size']|d('50')}}"
+                    },
+                    "dataDisks" : [
+{% for vol in instance['volumes']|default([]) %}
+                        {
+                            "caching" : "None",
+                            "createOption" : "Empty",
+                            "lun" : "{{loop.index}}",
+                            "name": "{{instancename}}-{{vol['device_name']}}",
+                            "diskSizeGB" : "{{vol['volume_size']}}"
+                        },
+{% endfor %}
+                    ]
+                },
+                "networkProfile" : {
+                    "networkInterfaces" : [
+                        {
+                            "id" : "[resourceId('Microsoft.Network/networkInterfaces','{{instancename}}-Interface')]"
+                        }
+                    ]
+                },
+                "diagnosticsProfile" : {
+                    "bootDiagnostics" : {
+                        "enabled" : "false",
+                        "storageUri" : "[concat(reference(concat('Microsoft.Storage/storageAccounts/', variables('storageAccountName')), '2016-01-01').primaryEndpoints.blob)]"
+                    }
+                }
+            }
+        },
+{% endfor %}
+{% endfor %}
```

Comment:
And here too:
```
{%   endfor %}
{% endfor %}
```

Reply:
Thanks @jkupferer, this file was copied from the configs/three-tier-app config, would you like me to make these changes there also?